libraries/Microsoft.Bot.Builder.Dialogs/Choices/Channel.cs

```diff
@@ -64,22 +64,8 @@ public static bool HasMessageFeed(string channelId)
         public static string GetChannelId(ITurnContext turnContext) => string.IsNullOrEmpty(turnContext.Activity.ChannelId)
             ? string.Empty : turnContext.Activity.ChannelId;
 
-        public class Channels
+        public class Channels : Connector.Channels

```

Comment:
Can you add a detailed comment here, and an obsolete attribute?

https://stackoverflow.com/questions/1759352/how-to-mark-a-method-as-obsolete-or-deprecated

Reply:
Let me know your thoughts on the commenting. They seem lackluster, but that's all I could think of at the time.